scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -591,6 +581,43 @@ <h6>No pinned variants</h6>
   </div>
 {% endmacro %}
 
+
+{% macro validation_macro(variant, case) %}
+ <form action="{{ url_for('cases.mark_validation', institute_id=variant.institute, case_name=case.display_name, variant_id=variant._id) }}" method="POST" accept-charset="utf-8" id="validation_form">
+  <select class="form-control form-control-sm" onchange="validate_action(this.value);" name="type">
+    {% for type in ('Not validated', 'True positive', 'False positive') %}
+      <option value="{{ type }}" {% if type == variant.validation %}selected{% endif %}>{{ type }}</option>
+    {% endfor %}
+  </select>
+ </form>
+
+<div class="modal fade" tabindex="-1" id="validationModal">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Positive validation notification</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <form action="{{ url_for('cases.mark_validation', institute_id=variant.institute, case_name=case.display_name, variant_id=variant._id) }}" method="POST" accept-charset="utf-8">
+        <input type="hidden" name="type" value="True positive"/>
+        <div class="modal-body">
+          <p>The variant is being marked as a true positive.</p>
+          <input class="form-check-input" type="checkbox" value="yes" name="notify_user" id="notify_user" checked>
+          <label class="form-check-label" for="notify_user">
+            Notify user which ordered verification
```

Comment:
```suggestion
            Notify user who ordered verification
```
Unless it's one of those function email boxes at work of course! 😉